common.c

```diff
@@ -314,19 +319,13 @@ get_config_file()
 	rtr_getenv_t real_getenv;
 	int olderrno;
 
-	if (!get_tracing_enabled())
```

Comment:
@riataman thanks for looking this over. `rtr_parse_config` does the `trace_disable()` then calls `get_config()` which in turn calls `get_config_file()`

I'd like to have put an assertion in. Any opinions on using assert()?